egs/cifar/v1/image/split_ocr_dir.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
Where is this script used?
I guess it should be in the `local` directory and it needs documentation.